libpretixsync/src/main/java/eu/pretix/libpretixsync/models/db/OrderPositionExtensions.kt

```diff
@@ -50,6 +50,15 @@ private fun parsePrice(json: JSONObject): BigDecimal? {
     }
 }
 
+private fun parseTaxCode(json: JSONObject): String? {
```

Comment:
`taxCode` still needs to be assigned using this function at the top of the file in `OrderPosition.toModel()`